StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -633,10 +633,14 @@ extension NativeFlowController: AccountPickerViewControllerDelegate {
         _ viewController: AccountPickerViewController,
         didSelectAccounts selectedAccounts: [FinancialConnectionsPartnerAccount],
         nextPane: FinancialConnectionsSessionManifest.NextPane,
-        customSuccessPaneMessage: String?
+        customSuccessPaneMessage: String?,
+        saveToLinkWithStripeSucceeded: Bool?
     ) {
         dataManager.linkedAccounts = selectedAccounts
         dataManager.customSuccessPaneMessage = customSuccessPaneMessage
+        if let saveToLinkWithStripeSucceeded {
+            dataManager.saveToLinkWithStripeSucceeded = saveToLinkWithStripeSucceeded
```

Comment:
this will activate some logic in another part of the code (the important part is passing on what happened)